package.json

```diff
@@ -40,8 +41,8 @@
     "typescript-plugin-inner-jsx": "^0.1.9"
   },
   "peerDependencies": {
-    "react": "^16.6.0",
-    "react-dom": "^16.x",
+    "react": "^16.8.0",
+    "react-dom": "^16.8",
     "styled-components": "^2.x"
```

Comment:
Any reason *not* to finally update styled components to v4?

I think sooner or later we need to do it as v2 has no future and we need to get on the latest here, otherwise adoption of this is already doomed (either people have no styled components yet, so we can bundle it already, or they have >= 4 which makes our peer dep failing).

Reply:
It's coming with the next PR. It's done already but I wanted to separate 2 these things. This PR has lots of changes already